libhb/qsv_common.c

```diff
@@ -1542,6 +1542,15 @@ int hb_qsv_profile_parse(hb_qsv_param_t *param, hb_qsv_info_t *info, const char
 
             case MFX_CODEC_HEVC:
                 profile = hb_triplet4key(hb_qsv_h265_profiles, profile_key);
+
+                /* HEVC10 supported starting from KBL/G6 */
+                if (profile->value == MFX_PROFILE_HEVC_MAIN10 &&
+                    qsv_hardware_generation(hb_get_cpu_platform()) < QSV_G6)
+                {
+                    hb_log("HEVC Main10 is not supported on this platform");
```

Comment:
good points: will fix first one and second: should yet try GUI (prefer CLI therefore these changes)

@sr55 would you have any time to try on KBL HW ? 
if so , I will think to share access to my test PC.